dts/bindings/ethernet/ethernet.yaml

```diff
@@ -12,3 +12,16 @@ properties:
       description: Specifies the MAC address that was assigned to the network device
     label:
       required: true
+    zephyr,random-mac-address:
+      type: boolean
+      required: false
+      description: |
+        Use a random MAC address generated when the driver is initialized.
+        Note that using this choice and rebooting a board may leave stale
+        MAC address in peers' ARP caches and lead to issues and delays in
+        communication.  (Use "ip neigh flush all" on Linux peers to clear
+        ARP cache.)
+
+        Its driver specific how the OUI octets are handled.
```

Comment:
`It's` or `It is` if we want to be pedantic.  Not worth changing if this isn't touched again.

Reply:
I was at the back of the CI queue so went ahead and fixed it. :)